client/data/promote-post/types.ts

```diff
@@ -56,6 +56,9 @@ export type CampaignStats = {
 	clicks_total: number;
 	spent_budget_cents: number;
 	deliver_margin_multiplier: number;
+	conversions_total?: number;
+	conversion_value?: number;
+	conversion_rate?: number;
```

Comment:
I'll follow this rules for the conversion endpoint :D

Reply:
@sbarbosa  this numbers will be returned as decimals right? and then I need to format them

Reply:
Yeah, I think we can assume that the format will be the same as the rest of the stats. We have already returned the total clicks and CTR from the DSP.

Reply:
> @sbarbosa this numbers will be returned as decimals right? and then I need to format them
I think we should return them in a way that they don't have to format in each clients if there are more than one!